pkg/flare/archive_win.go

```diff
@@ -82,6 +96,76 @@ func zipTypeperfData(tempDir, hostname string) error {
 	return nil
 }
 
+// zipEventLogs exports all the Windows event logs.
+func zipEventLogs(tempDir, hostname string) error {
+	// Enumerate all event channels registered.
+	eventLogChannels, err := windowsevent.EnumerateChannels()
+
+	if err != nil {
+		log.Warnf("could not list windows event log channels: %v", err)
+		return err
+	}
+
+	log.Infof("found event log channels: %v", eventLogChannels)
```

Comment:
Should this be in `debug` ?

Reply:
It need to appear in the final log file in flare. Info level is correct.